src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,16 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines. Please remove the unnecessary empty lines before the class declaration.